include/oc_ri.h

```diff
@@ -342,6 +360,11 @@ typedef struct oc_event_callback_s
   void *data;                       ///< data for the callback
 } oc_event_callback_t;
 
+/**
+ * @brief CoAP status code string
+ */
+extern const char *cli_status_strs[];
```

Comment:
You shouldn't need this since `oc_status_to_str` is available, right?

Reply:
oops, right. 
fixed.